pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -254,6 +256,15 @@ func (h *NetworkUtilsHandler) GenerateRandomMac() (net.HardwareAddr, error) {
 	return net.HardwareAddr(append(prefix, suffix...)), nil
 }
 
+func (h *NetworkUtilsHandler) NeighDelete(iface, macAddress string) error {
+	// Todo: (schSeba) Change this with netlink function when available
+	// PR: https://github.com/vishvananda/netlink/issues/439
```

Comment:
This is an issue, not a PR. Did you already create the PR?

Reply:
you are right my bad I will fix its an issue I didn't create a PR for that.